Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/FareProduct.hs

```diff
@@ -38,6 +38,13 @@ findAllFareProductForVariants ::
   m [Domain.FareProduct]
 findAllFareProductForVariants (Id merchantId) area = findAllWithKV [Se.And [Se.Is BeamFP.merchantId $ Se.Eq merchantId, Se.Is BeamFP.area $ Se.Eq area]]
 
+findAllFareProductForFlow ::
```

Comment:
Can you use this in `getFarePolicy`?

Reply:
findAllFareProductForFlow we want only fare for rental flow right so how can we do same with getFarePolicy?